test/test_models.py

```diff
@@ -193,7 +192,7 @@ def _check_fx_compatible(model, inputs):
 # the _test_*_model methods.
 _model_params = {
     'inception_v3': {
-        'input_shape': (1, 3, 299, 299)
+        'input_shape': (1, 3, 299, 299),
```

Comment:
I realize this is due to previous changes that were reverted, but in general we try to avoid unrelated changes, as it obfuscates git blame. Same for the removal of `import sys` above, which on his own is actually relevant (and doesn't hurt git blame), but having lots of those in a single PR makes review more difficult. Would you mind reverting those changes?

Reply:
I agree i'll revert these!